wpilib/wpilib/command/subsystem.py

```diff
@@ -138,36 +133,36 @@ def getCurrentCommand(self):
         """
         return self.currentCommand
 
-    def __str__(self):
-        return self.getName()
+    def getCurrentCommandName(self):
+        """
+        Returns the current command name, or empty string if no current command.
 
-    def getName(self):
-        """Returns the name of this subsystem, which is by default the class
-        name.
-        
-        :returns: the name of this subsystem
+        :returns: the current command name
+        """
+        currentCommand = self.getCurrentCommand()
+        if currentCommand is not None:
+            return currentCommand.getName()
+        return ""
+
+    def addChild(self, child, name=None):
```

Comment:
This doesn't match the order of the optional `name` upstream. This is a new API, so we could deviate from upstream I suppose. Do we want to here, @virtuald?

Reply:
Optional arguments have to go last, so deviation is fine.